suite-common/formatters/src/formatters/prepareDateTimeFormatter.ts

```diff
@@ -0,0 +1,15 @@
+import { A, pipe } from '@mobily/ts-belt';
+
+import { makeFormatter } from '../makeFormatter';
+import { FormatterConfig } from '../types';
+import { prepareDateFormatter } from './prepareDateFormatters';
+import { prepareTimeFormatter } from './prepareTimeFormatter';
+
+export const prepareDateTimeFormatter = (config: FormatterConfig) =>
+    makeFormatter<Date | number | null, string | null>(value => {
+        if (!value) return null;
+        const DateFormatter = prepareDateFormatter(config);
+        const TimeFormatter = prepareTimeFormatter(config);
+
+        return pipe([TimeFormatter.format(value), DateFormatter.format(value)], A.join(', '));
```

Comment:
Isn't pipe redundant here? 
```suggestion
        return [TimeFormatter.format(value), DateFormatter.format(value)].join(', ')
```

Reply:
It doesn't really affect anything performance-wise and is easily extendable + if we want to introduce more functional code from workshop we might as well use it.

Reply:
If you believe that others will be potentially added, leave it like it is. But since this is only a concatenation of the results of two other matters, I don't see a reason for the pipe.

Reply:
You never know. But since it really doesn't matter and it's just a preference, I guess it's up to us as a team to have some conventions. Since we all have a better understanding about this and there was a proposition from @Nodonisko to use it even before the workshop so that we get used to it and don't even have to think about it twice, I think it doesn't really matter. 

Same argument can be applied to say I don't really see a reason why not to use it, as we want to start using it and unify the code a little bit.